common/src/main/java/io/homeassistant/companion/android/common/sensors/AndroidOsSensorManager.kt

```diff
@@ -0,0 +1,80 @@
+package io.homeassistant.companion.android.common.sensors
+
+import android.content.Context
+import android.os.Build
+import io.homeassistant.companion.android.common.R as commonR
+
+class AndroidOsSensorManager : SensorManager {
+
+    companion object {
+        val osVersion = SensorManager.BasicSensor(
+            "android_os_version",
+            "sensor",
+            commonR.string.basic_sensor_name_android_os_version,
+            commonR.string.sensor_description_android_os_version,
+            "mdi:android",
+            entityCategory = SensorManager.ENTITY_CATEGORY_DIAGNOSTIC
+        )
+        val osSecurityPatch = SensorManager.BasicSensor(
+            "android_os_security_patch",
+            "sensor",
+            commonR.string.basic_sensor_name_android_os_security_patch,
+            commonR.string.sensor_description_android_os_security_patch,
+            "mdi:security",
+            entityCategory = SensorManager.ENTITY_CATEGORY_DIAGNOSTIC
+        )
+    }
+
+    override fun docsLink(): String {
+        return "https://companion.home-assistant.io/docs/core/sensors#android-os-sensors"
+    }
+
+    override val name: Int
+        get() = commonR.string.sensor_name_android_os
+
+    override suspend fun getAvailableSensors(context: Context): List<SensorManager.BasicSensor> {
+        return listOf(osVersion, osSecurityPatch)
+    }
+
+    override fun requiredPermissions(sensorId: String): Array<String> {
+        return arrayOf()
+    }
+
+    override fun hasSensor(context: Context): Boolean {
+        return true
+    }
```

Comment:
this is actually the default value, we only need to override it if the value will be `false` :)

https://github.com/home-assistant/android/blob/master/common/src/main/java/io/homeassistant/companion/android/common/sensors/SensorManager.kt#L138